database/neo4j/neo4j.go

```diff
@@ -0,0 +1,231 @@
+package neo4j
+
+import (
+	"fmt"
+	"github.com/prometheus/common/log"
```

Comment:
Don't pull in another logging package...

Reply:
Removed, didn't even see that